src/main/java/network/kngsgaming/minecraft/EnchantEvents.java

```diff
@@ -37,9 +37,8 @@ public EnchantEvents( VanillaEnchants plugin) {
             plugin.printToConsole(ChatColor.RED + "ERROR: could not get limits from config");
         }
         try {
-            debug = plugin.config.getString("debug").toLowerCase().equals("true");
+            debug = plugin.config.getString("debug").equalsIgnoreCase("true");
```

Comment:
This is great! I think if you edit your pull request or submit a new one with this that would be awesome. I didn't know about equalsIgnoreCase(). I didn't see your request until after I made some changes a little while ago.